### PR TITLE
I added Azerbaijani Dataset

### PR DESCRIPTION
**Pull request by ferid333**
I added Azerbaijani Dataset

### CODE REVIEW

**Comment by lk-geimfari** [comment]
Hi! Please see https://github.com/lk-geimfari/mimesis/tree/master/mimesis/datasets/locale_template

**Comment by ferid333** [comment]
Sorry I will fix

**Comment by ferid333** [comment]
Hello. Everything works fine but there is problem related to language. In our language if a person is female, surname of her should be added 'a' at the end. For example, Afaq Qurbanov**a** but for males Farid Qurbanov

**Comment by lk-geimfari** [comment]
> Hello. Everything works fine but there is problem related to language. In our language if a person is female, surname of her should be added 'a' at the end. For example, Afaq Qurbanov**a** but for males Farid Qurbanov

Same in Russian. Please, refer to `ru` locale.

**Comment by ferid333** [comment]
I just update the surnames, everything is fine now and working

**Comment by lk-geimfari** [comment]
@ferid333 We're almost done. Please fix the tests and add locale to the table of locales: https://github.com/lk-geimfari/mimesis/blob/master/docs/locales.rst#supported-locales

**Comment by ferid333** [comment]
@lk-geimfari Error is related  to this function:  Where SUPPORTED_LOCALES_COUNT does not match
def test_locale_enum():
>       assert len(list(Locale)) == SUPPORTED_LOCALES_COUNT
E       AssertionError: assert 48 == 47
E        +  where 48 = len([<Locale.AR_AE: 'ar-ae'>, <Locale.AR_DZ: 'ar-dz'>, <Locale.AR_EG: 'ar-eg'>, <Locale.AR_JO: 'ar-jo'>, <Locale.AR_OM: 'ar-om'>, <Locale.AR_KW: 'ar-kw'>, ...])
E        +    where [<Locale.AR_AE: 'ar-ae'>, <Locale.AR_DZ: 'ar-dz'>, <Locale.AR_EG: 'ar-eg'>, <Locale.AR_JO: 'ar-jo'>, <Locale.AR_OM: 'ar-om'>, <Locale.AR_KW: 'ar-kw'>, ...] = list(Locale)

**Comment by lk-geimfari** [comment]
> @lk-geimfari Error is related to this function: Where SUPPORTED_LOCALES_COUNT does not match def test_locale_enum():
> 
> > ```
> >   assert len(list(Locale)) == SUPPORTED_LOCALES_COUNT
> > ```
> 
> E AssertionError: assert 48 == 47 E + where 48 = len([<Locale.AR_AE: 'ar-ae'>, <Locale.AR_DZ: 'ar-dz'>, <Locale.AR_EG: 'ar-eg'>, <Locale.AR_JO: 'ar-jo'>, <Locale.AR_OM: 'ar-om'>, <Locale.AR_KW: 'ar-kw'>, ...]) E + where [<Locale.AR_AE: 'ar-ae'>, <Locale.AR_DZ: 'ar-dz'>, <Locale.AR_EG: 'ar-eg'>, <Locale.AR_JO: 'ar-jo'>, <Locale.AR_OM: 'ar-om'>, <Locale.AR_KW: 'ar-kw'>, ...] = list(Locale)

Yeah, I know. Contributors must increment the `SUPPORTED_LOCALES_COUNT` when adding a locale.

**Comment by ferid333** [comment]
> > @lk-geimfari Error is related to this function: Where SUPPORTED_LOCALES_COUNT does not match def test_locale_enum():
> > > ```
> > >   assert len(list(Locale)) == SUPPORTED_LOCALES_COUNT
> > > ```
> > 
> > 
> > E AssertionError: assert 48 == 47 E + where 48 = len([<Locale.AR_AE: 'ar-ae'>, <Locale.AR_DZ: 'ar-dz'>, <Locale.AR_EG: 'ar-eg'>, <Locale.AR_JO: 'ar-jo'>, <Locale.AR_OM: 'ar-om'>, <Locale.AR_KW: 'ar-kw'>, ...]) E + where [<Locale.AR_AE: 'ar-ae'>, <Locale.AR_DZ: 'ar-dz'>, <Locale.AR_EG: 'ar-eg'>, <Locale.AR_JO: 'ar-jo'>, <Locale.AR_OM: 'ar-om'>, <Locale.AR_KW: 'ar-kw'>, ...] = list(Locale)
> 
> Yeah, I know. Contributors must increment the `SUPPORTED_LOCALES_COUNT` when adding a locale.

Where is that ?

**Comment by lk-geimfari** [comment]
@ferid333 https://github.com/lk-geimfari/mimesis/blob/4212746d62d7f73a3cd85a0c3c2599a16928b3e2/tests/test_locales.py#L8

**Comment by lk-geimfari** [comment]
@ferid333 Please, resolve conflicts.

**Comment by lk-geimfari** [comment]
@ferid333 Some tests are still failing. Please, fix them. Thanks!

**Comment by ferid333** [comment]
@lk-geimfari  What error is related to ? CHECK=1 bash scripts/format.sh
Running lint check
would reformat /home/runner/work/mimesis/mimesis/mimesis/datasets/int/code.py
would reformat /home/runner/work/mimesis/mimesis/tests/test_locales.py
would reformat /home/runner/work/mimesis/mimesis/mimesis/datasets/int/finance.py

Oh no! 💥 💔 💥
3 files would be reformatted, 111 files would be left unchanged.
make: *** [Makefile:20: format] Error 1

**Comment by lk-geimfari** [comment]
@ferid333 There are code style issues. You can fix them using `make format`

**Comment by ferid333** [comment]
> @ferid333 There are code style issues. You can fix them using `make format`

I made

**Comment by ferid333** [comment]
@lk-geimfari  What can be reason for that : @codecov
codecov
/ codecov/patch
mimesis/enums.py#L81
Added line #L81 was not covered by tests

File: mimesis/datasets/az/food.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  "noxud",
+  "acıbibər",
+  "kartof",
+  "balqabaq",
+  "qabaq",
+  "ispanaq",
+  "pomidor"
+]
+}
+
```
Comment (lk-geimfari) [inline mimesis/datasets/az/food.json:154]: Please, remove whitespaces.

File: mimesis/datasets/az/text.json [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+    "təşəkkür",
+    "bəli",
+    "xeyr",
+    "həyat",
+     "paltar",
+    "pul",
+    "avtobus",
+    "maşın"
+]
+}
```
Comment (lk-geimfari) [inline mimesis/datasets/az/text.json:276]: Same here.

**Review by lk-geimfari — CHANGES_REQUESTED** [review]

File: pyproject.toml [hunk 3]
```diff
@@ -54,6 +54,7 @@ exclude = [
 python = "^3.10"
 factory-boy = { version = "^3.3.0", optional = true }
 pytest = { version = "^7.2", optional = true }
+isort = "^5.13.2"
```
[the remark this answers is not in the document]
Comment (ferid333) [reply pyproject.toml:57]: @lk-geimfari I fixed. Thanks

**Comment by lk-geimfari** [comment]
@ferid333 @ferid333 You forgot to lock dependencies after removing `isort`. Please lock deps, using `poetry lock`.

**Comment by lk-geimfari** [comment]
@ferid333 We’re done. I’ll merge the PR later after a detailed review on my machine. Thank you!

**Comment by ferid333** [comment]
> @ferid333 We’re done. I’ll merge the PR later after a detailed review on my machine. Thank you!

Thank you

**Comment by ferid333** [comment]
Hello! Could you merge ?

**Comment by lk-geimfari** [comment]
@ferid333 Sorry, I've been busy as hell these past few months.

**Pull request merged**